.github/workflows/release.yml

```diff
@@ -27,16 +27,16 @@ jobs:
           draft: ${{ github.ref_name != 'main'}}
           input: "proto/composerapis"
           buf_token: ${{ secrets.BUF_TOKEN }}
-      - name: "Upload Sequencer API Protos"
+      - name: "Upload Primitive API Protos"
```

Comment:
Note: I will need to manually push some of these crates for the first time in order to tests to pass. I will push once I have review that covers the protos